tableauserverclient/server/endpoint/metadata_endpoint.py

```diff
@@ -1,12 +1,43 @@
 from .endpoint import Endpoint, api
-from .exceptions import GraphQLError
-
+from .exceptions import GraphQLError, InvalidGraphQLQuery
 import logging
 import json
 
 logger = logging.getLogger('tableau.endpoint.metadata')
 
 
+def is_valid_paged_query(parsed_query):
```

Comment:
Are these methods outside of the Metadata class because they are static?

Reply:
Yup, I thought about `staticmethods` but they're really just internal helpers for parsing the json, so I moved them to module functions